docs/classifiers/multilayer-perceptron.md

```diff
@@ -77,6 +77,20 @@ Returns the underlying neural network instance or `null` if untrained:
 public network() : Network|null
 ```
 
+Export a Graphviz "dot" encoding of the neural network architecture.
+```php
+public exportGraphviz() : Encoding
+```
+
+```php
+use Rubix\ML\Helpers\Graphviz;
+use Rubix\ML\Persisters\Filesystem;
+
+$dot = $estimator->exportGraphviz();
+
+Graphviz::dotToImage($dot)->saveTo(new Filesystem('network.png'));
+```
```

Comment:
do you want to add an image here? like the one in this PR?

Reply:
Yes great idea